DittoDataBrowser/src/main/java/live/ditto/dittodatabrowser/Documents.kt

```diff
@@ -97,12 +97,6 @@ fun Documents(collectionName: String, isStandAlone: Boolean) {
                     text = "No Docs",
                     textAlign = TextAlign.Start,
                     color = Color.Blue,
-                    modifier = Modifier
```

Comment:
Why was this removed?

Reply:
It's on the "No Docs" text which means the user cannot click on the text to see the documents. There is nothing to modify when it shows "No Docs".